package.json

```diff
@@ -13,6 +13,7 @@
 	"types": "./lib/src/index.d.ts",
 	"dependencies": {
 		"@babel/runtime": "^7.4.4",
+		"msal": "^1.4.4",
```

Comment:
Will the msal version change cause any problem ?

Reply:
I don't believe it will as it was ^1.0.0 which will resolve to 1.4.4 if the consumer hasn't locked to a specific version in their project, or work from a different copy if they have. I can set it to 1.0.0 if you think we should give consumers "more freedom"